src/FluentNHibernate.Testing/FluentNHibernate.Testing.csproj

```diff
@@ -1,474 +1,61 @@
-    <Compile Include="MappingModel\Output\XmlOneToManyWriterTester.cs" />
-    <Compile Include="MappingModel\Output\XmlOneToOneWriterTester.cs" />
-    <Compile Include="MappingModel\Output\XmlPropertyWriterTester.cs" />
-    <Compile Include="MappingModel\Output\XmlSetWriterTester.cs" />
-    <Compile Include="MappingModel\Output\XmlVersionWriterTester.cs" />
-    <Compile Include="MappingModel\Output\XmlWriterContainerTester.cs" />
-    <Compile Include="PersistenceModelTests\JoinPersistenceModelTests.cs" />
-    <Compile Include="PersistenceModelTests\SeparateSubclassVisitorFixture.cs">
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="PersistenceModelTests\SubclassPersistenceModelTests.cs" />
-    <Compile Include="Serialization\SerializationFixture.cs" />
-    <Compile Include="Testing\PersistenceSpecificationExtensionsSpecs.cs" />
-    <Compile Include="Testing\PersistenceSpecificationTester.cs" />
-    <Compile Include="Testing\PersistenceSpecificationTransactionTest.cs" />
-    <Compile Include="Testing\SessionSourceConfigurationTester.cs" />
-    <Compile Include="Specification.cs" />
-    <Compile Include="SpecificationExtensions.cs" />
-    <Compile Include="Debugging.cs" />
-    <Compile Include="DomainModel\Employees.cs" />
-    <Compile Include="DomainModel\Music.cs" />
-    <Compile Include="MappingModel\AttributeStoreTester.cs" />
-    <Compile Include="MappingModel\ClassMappingTester.cs" />
-    <Compile Include="MappingModel\HibernateMappingTester.cs" />
-    <Compile Include="MappingModel\MappingTestingExtensions.cs" />
-    <Compile Include="Testing\Values\ListSpecs.cs" />
-    <Compile Include="Testing\Values\ReferenceBagSpec.cs" />
-    <Compile Include="Testing\Values\ReferenceListSpecs.cs" />
-    <Compile Include="Testing\Values\ReferencePropertySpecs.cs" />
-    <Compile Include="Testing\Values\PropertySpecs.cs" />
-    <Compile Include="Testing\Values\Entities.cs" />
-    <Compile Include="Utils\ReflectionHelperTests.cs" />
-    <Compile Include="Testing\XmlWriterTestHelper.cs" />
-    <Compile Include="Utils\TypeReferenceEqualityTests.cs" />
-    <Compile Include="Visitors\ComponentColumnPrefixVisitorSpecs.cs" />
-    <Compile Include="Visitors\ComponentReferenceResolutionVisitorSpecs.cs" />
-    <Compile Include="Visitors\RelationshipPairingVisitorSpec.cs" />
-    <Compile Include="Visitors\RelationshipPairingVisitorSpecSupportClasses.cs" />
-    <Compile Include="Visitors\RelationshipPairingSpec.cs" />
-    <Compile Include="Xml\MappingXmlTestHelper.cs" />
+    <ProjectReference Include="..\FluentNHibernate\FluentNHibernate.csproj" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\FluentNHibernate\FluentNHibernate.csproj">
-      <Project>{529145D5-6CC3-4016-89B5-826563024AF1}</Project>
-      <Name>FluentNHibernate</Name>
-    </ProjectReference>
+
+  <ItemGroup>    
+    <PackageReference Include="EntityFramework" Version="6.2.0" />

```

Comment:
Do we need a reference to EntityFramework?

Reply:
Not at the beginning. So I think we can remove the dependency without problems. 👍